src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
+
+  if (days >= 7) {
+    sale = 50;
```

Comment:
Avoid using literals for the discount values and minimum days. Define constants at the beginning of the script to make the code more readable and maintainable, for example, `const LONG_TERM_DISCOUNT = 50;` and `const LONG_TERM_THRESHOLD = 7;`.